tensornetwork/contractors/custom_path_solvers/example.py

```diff
@@ -0,0 +1,48 @@
+
+
+import numpy as np
```

Comment:
Be sure to include the copyright notice at the top of every file.

Reply:
Thanks, should be taken care of.